[17.0][IMP] sign_oca: add kanban/activity view on sign.oca.request.signer

jguenat opened the pull request:
backport of #111

jguenat commented:
do not merge, there is an issue with the backport of the kanban vue

jguenat commented:
@victoralmau @pedrobaeza 
Backport of merged #111 
I modified kanban view for 17.0 compatibility and changed systray action call name to incorporate improvement made during V18 migration.

victoralmau commented on sign_oca/views/sign_oca_request.xml:
```diff
@@ -365,14 +366,89 @@
                         <field name="longitude" />
                     </group>
                 </sheet>
+                <chatter />
```
```suggestion
<div class="oe_chatter">
                     <field name="message_follower_ids"/>
                     <field name="activity_ids"/>
                     <field name="message_ids"/>
                </div>
```

victoralmau commented on sign_oca/views/sign_oca_request.xml:
```diff
@@ -365,14 +366,89 @@
         <field name="name">Sign Request Signers</field>
         <field name="res_model">sign.oca.request.signer</field>
-        <field name="view_mode">tree,form</field>
+        <field name="view_mode">kanban,list,form,activity</field>
```
```suggestion
        <field name="view_mode">kanban,tree,form,activity</field>
```

victoralmau reviewed (changes requested):
Changes related to `systray_service.esm.js` should go in a separate commit to avoid confusion.

jguenat committed:
[IMP] sign_oca: improve request.signer views
- add kanban & search view
- enable activity : add chatter on form view
- sign button primary

jguenat committed:
[IMP] sign_oca: systray action name
- this is a backport of an improvement done during 18.0 migration

jguenat commented:
@victoralmau  thanks for the review, modifications made

etobella reviewed (approved):
/ocabot merge patch
